core/server/common/src/main/java/alluxio/cli/ValidateEnv.java

```diff
@@ -222,7 +222,7 @@ private static boolean validateLocal(String target, String name, String[] args)
     Collection<ValidationTask> tasks = TARGET_TASKS.get(target);
     for (ValidationTask task: tasks) {
       String taskName = TASKS.get(task);
-      if (name != null && !taskName.startsWith(name)) {
+      if (name != null && !taskName.startsWith(name) || task.shouldSkip()) {
```

Comment:
move this to after printing out "Validating %s" so user know which task is skipped:
```
       System.out.format("Validating %s...", taskName);
       if (task.shouldSkip()) {
           System.out.println("Skipped");
           continue;
       }
```

Reply:
Done.
Previously, if a task was skipped, it would also be printed.